src/App.tsx

```diff
@@ -1,27 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+import 'bulma/css/bulma.css';
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [isAllGoodsVisible, setAllGoodsVisible] = useState(false);
+  const [isFiveGoodsVisible, setFiveGoodsVisible] = useState(false);
+  const [isRedGoodsVisible, setRedGoodsVisible] = useState(false);
```

Comment:
if you'll have 10 separate functions for getting data from api, will you create 10 separate variables in state?

try to complete this task without creating additional variables in state, only `goods` will be enough